modules/deploy/21-couchdb-persistentvolume-us-east-2b.erb

```diff
@@ -4,8 +4,14 @@ apiVersion: v1
 kind: PersistentVolume
 metadata:
   name: couchdb-us-east-2b-pv
+  annotations:
+<% if ENV["TF_VAR_cluster_name"].start_with? "stg." or ENV["TF_VAR_cluster_name"].start_with? "prd." %>
+    backup.kubernetes.io/deltas: "PT15M PT60M PT4H PT24H"
+<% else %>
+    backup.kubernetes.io/deltas: "PT5M PT15M PT45M"
+<% end %> 
   labels:
-    couchdb-pv: us-east-2b
+    couchdb-pv: us-east
```

Comment:
I don't think this change is right. I'm pretty sure we need to refer to a specific volume in a specific AZ, otherwise we can't restore from backup.

It's possible I'm wrong. I guess you'll find out when you test backup/restore with the new arrangement :).